interface/wx/filehistory.h

```diff
@@ -15,6 +32,8 @@
     required in an MDI application, where the file history should appear on
     each MDI child menu as well as the MDI parent frame.
 
+    By default, the menu item label style will be @c wxFH_HIDE_CURRENT_PATH.
```

Comment:
```suggestion
    By default, the menu item label style is @c wxFH_HIDE_CURRENT_PATH,
    meaning that paths are only shown in the menu items if they differ.
    This can be changed using SetMenuLabelStyle().
```